### DISCUSSION TITLE
In which Avalonia version is the new FilePicker API?

### DISCUSSION THREAD

**Discussion by 0x90d** [AvaloniaUI/Avalonia#10029] [Q&A] [answered]
The new filepicker api was merged in July 2022. I'm currently on `0.10.18` which was released in August 2022. But `FolderPickerOpenOptions` and `StorageProvider` don't exist. Do I have to add some new nuget package or something?

![image](https://user-images.githubusercontent.com/46010672/213672129-a9b2b66a-d948-45cf-a661-52d8b16b5e71.png)

**Comment by timunie** [comment, marked as answer]
It's only a available in the upcoming version 11.0. you can test it using the preview versions of 11.0. 

Be careful, there are some breaking changes to expect when upgrading from 0.10.18.
